split.Console/Models/Deck.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+namespace split.Console.Models
+{
+    public class Deck
+    {
+        Card[] Cards;
+        int CardsIndex;
+        const int CARDSINDECK = 52;
+
+        public Deck(int numberDecks)
+        {
+            Cards = new Card[numberDecks * CARDSINDECK];
+            CardsIndex = 0;
+
+            for (int i = 0; i < numberDecks; i++)
+            {
+                foreach (SuitType suit in Enum.GetValues(typeof(SuitType)))
+                {
+                    foreach (RankType rank in Enum.GetValues(typeof(RankType)))
+                    {
+                        Cards[CardsIndex++] = new Card(suit, rank);
+                    }
+                }
+            }
+            Shuffle();
```

Comment:
Do you think having the shuffle method directly shuffle the cards member is okay or should it take it as a parameter?